app/static/js/admin/event/wizard/vue-app.js

```diff
@@ -176,6 +179,10 @@ var app = new Vue({
         },
         addTicket: function (ticketType) {
             var ticket = _.cloneDeep(TICKET);
+            ticket.sales_start_date = moment().tz(this.event.timezone).format('MM/DD/YYYY');
```

Comment:
@shubham-padia while storing there is no need to resolve the timezone. Store the plain date/time without any timezone information. Then resolve the timezone in the backend if and when required by referencing the timezone from the event. That way, we'll always use the timezone of the event and we don't have to change the timezone data in all the tickets when the event timezone changes.